apps/server/src/modules/tldraw/controller/tldraw.controller.ts

```diff
@@ -1,7 +1,7 @@
 import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { Controller, Delete, ForbiddenException, HttpCode, NotFoundException, Param } from '@nestjs/common';
 import { ApiValidationError } from '@shared/common';
-import { TldrawService } from '../service/tldraw.service';
+import { TldrawService } from '../service';
```

Comment:
why do you use the service directly? there should be a uc in between, which should be responsible for authorisation, which appears to be missing entirely. Unless the whole deployment is never accessible by a user, this of course needs to be fixed before the feature can be activated.

Reply:
as discussed, not in the scope of this PR so will be done in a separate one